cmd/kev/cmd/render.go

```diff
@@ -103,19 +102,28 @@ func runRenderCmd(cmd *cobra.Command, _ []string) error {
 	}
 
 	setReporting(verbose)
-	displayCmdStarted(cmdName)
-
-	workingDir, err := os.Getwd()
-	if err != nil {
-		return displayError(err)
-	}
-
-	log.DebugTitlef("Output format: %s", format)
-	if err := kev.Render(workingDir, format, singleFile, dir, envs, nil); err != nil {
-		return displayError(err)
-	}
-
-	os.Stdout.Write([]byte("\n"))
-
-	return nil
+	// displayCmdStarted(cmdName)
```

Comment:
is this section still needed?

Reply:
Nope, its legacy - will come out in the wash 🙂